stab_controller_multi_lqr_node.cpp

```diff
@@ -0,0 +1,416 @@
+/* © Ascento; focusproject 2017/2018; ASL ETH Zurich
+___   _____ _____________   _____________
+/   | / ___// ____/ ____/ | / /_  __/ __  |
+/ /| | \__ \/ /   / __/ /  |/ / / / / / / /
+/ ___ |___/ / /___/ /___/ /|  / / / / /_/ /
+/_/  |_/____/\____/_____/_/ |_/ /_/  \____/
+Author: Lionel Gulich <lgulich@student.ethz.ch> - February 2018
+--------------------------------------------------------------------------------
+Code provided under the GNU General Public Licence GNU GPLv3
+--------------------------------------------------------------------------------
+This code provides the framework for the stabilisation control of the sara
+prototype. It uses multiple lqr controllers derived from different equilibrium
+positions of the system. Between the positions the controllers are interpolated.
+For read in of the k matrix array the matrices should be stored in "/resources"
+as a .csv file and named as "k_matrix_0", "k_matrix_1", ...
+For read in of the imu_offset scaler array the vectors should be stored
+in "/resources" as a .csv file and named as "imu_offset_0", "imu_offset_1", ...
+--------------------------------------------------------------------------------
+TODO: is double necessary in matrices or would float be enough?
```

Comment:
I would recommend doubles. Unless you are really diving into low level optimizations the difference between doubles and floats on runtime is generally minimal